pymove/utils/data_augmentation.py

```diff
@@ -0,0 +1,278 @@
+import numpy as np
+import pandas as pd
+
+from pymove.utils.constants import (
+    DATETIME,
+    LABEL,
+    LATITUDE,
+    LOCAL_LABEL,
+    LONGITUDE,
+    TID,
+    TRAJ_ID,
+    TRAJECTORY,
+)
+from pymove.utils.log import progress_bar
+
+
+def generate_trajectories_df(df_, label_local=LOCAL_LABEL):
+    """
```

Comment:
Codacy found an issue: [1 blank line required between summary line and description (found 0) (D205)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=6419254)